### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by paweltalar**

### CODE REVIEW

**Review by JoachimDahlgren — DISMISSED** [review]

**Review by eric-murray — APPROVED** [review]

**Comment by eric-murray** [comment]
Last call for any comments on this proposed documentation. If no comments are received by 17:00 GMT today, I'll merge the pull request.

File: documentation/API_documentation/Connectivity_API.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+This API makes possible to query the connectivity status of user equipment.
+
+## 1\. Introduction
+<img src="resources/DeviceStatus_Connectivity_diagram.png">
+
+## 2\. Quick Start
+DeviceStatus API v0.2.0 exposes only one endpoint, which can be used to check connectivity status for user equipment. To do so user has to pass following parameters in request body:
+1. **ueId** - an object with four fields, each of them make possible to pass ueId in different format: externalId [1], msisdn, ipv4Addr and ipv6Add
+2. **eventType** - string value which determines the connectivity event to check its status, for now only "UE_ROAMING_STATUS" is supported.
+
```
Comment (monamok) [inline documentation/API_documentation/Connectivity_API.md:11]: Just as a comment, uePort isn't a required input parameter and user doesn't "have to" send it but maybe we could mention it here as an optional.

**Review by eric-murray — DISMISSED** [review]

**Review by sachinvodafone — APPROVED** [review]

**Review by bigludo7 — DISMISSED** [review]
Good for me.
Thanks

**Comment by ToshiWakayama-KDDI** [comment]
Sorry for the late comment.  4.1 End point reads "Following table defines API endpoints of exposed REST based for QoD throughput management operations."  I wonder why "QoD throughput management operations" is hear.  Should this be changed to Device Status query operations or something?  

Thanks.

**Comment by eric-murray** [comment]
@ToshiWakayama-KDDI
This was a copy / paste error from the QoD API documentation. Now fixed.

**Comment by ToshiWakayama-KDDI** [comment]
Got it.  Thank you.

**Comment by eric-murray** [comment]
Now merged

**Pull request merged**